ui/src/cohortReview/cohortReviewList.tsx

```diff
@@ -53,8 +76,8 @@ class ReviewListItem {
     return this.pending?.size ?? this.review?.size;
   }
 
-  created() {
-    return this.pending?.created ?? this.review?.created;
+  lastModified() {
+    return this.pending?.lastModified ?? this.review?.cohort?.lastModified;
```

Comment:
This `lastModified` shows the review last modified, with a fallback to the review's cohort revision last modified. Is that correct?

Reply:
No, this object is to hold either pending reviews that are being created or actually existing reviews. Then they can be displayed in the list without knowing which is which.